director-configure-blobstore.html.md.erb

```diff
@@ -72,3 +72,23 @@ The Director and the Agents can use an S3 compatible blobstore. Here is how to c
         bucket_name: test-bosh-bucket
         host: objects.dreamhost.com
     ```
+
+---
+## <a id="gcs"></a> GCS
+
+The Director and the Agents can use GCS blobstore. Here is how to configure it:
```

Comment:
expand GCS to Google Cloud Storage (GCS) here. It's a less of a household name than S3.